common/prettify.pm

```diff
@@ -1662,13 +1662,15 @@ sub Config_Handler ($)
             my $revision = $totals->{GIT_REVISIONS}[0];
             print "Matched GIT url to revision $revision\n";
             $totals->{GIT_CHECKEDOUT_ACE} = $revision;
+            $commits{'GIT_COMMIT_ACE'} = $revision;
         }
         elsif ($url =~ m/(git|https):\/\/.*\/OpenDDS\.git/i)
```

Comment:
```suggestion
        elsif ($url =~ m/(git@|https:\/\/).*\/OpenDDS\.git/i)
```
These are the only two valid URL syntax styles for github.com, which OpenDDS is hosted at for the moment. I was having trouble testing because I was using git@ and not git:// (which isn't a thing). Other than this change, this PR looks good & ready to go.

Reply:
There are a number of valid URL styles and schemes for git.  `git://` is definitely one of them.  https://git-scm.com/docs/git-fetch#_git_urls

Reply:
Might want to make this change for the ACE_TAO URL as well? @mitza-oci any thoughts?

Reply:
Yeah, sorry, I modified my comment to account for that after I looked it up.